### ISSUE TITLE
analysis plan steps 

### ISSUE THREAD

**Issue by aromatic-toast** [aromatic-toast/IMDB_Sentiment_Analysis#1]
- [x] preprocess/clean up the raw text

- [x] train a doc2vec model on train set

- [x] infer the document vectors for validation & test sets 

- [x] extract the doc vectors for validataion & test and store as 2 separate csvs

- [x] make BOW & TFIDF representations using gensim of the train set 

- [x] try out 3 different classifiers on each vector representation on the validation set
